public/index.html

```diff
@@ -19,13 +19,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Visual Object Tagging Tool (VoTT) v%REACT_APP_VERSION%</title>
   </head>
   <body>
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
+    <div id="appInfo">VoTT v%REACT_APP_VERSION%, commit=%REACT_APP_COMMIT_SHA%</div>
```

Comment:
How do these %variables% work since they are outside of the React render pipeline? Webpack magic?

Reply:
yep, when the app gets built, it does a static replace of these.  I put the link elsewhere, but the 'rules' are that your env var has to start w/ `REACT_APP_` :crystal_ball: